frame/fee-market/src/lib.rs

```diff
@@ -138,7 +134,7 @@ pub mod pallet {
 		Blake2_128Concat,
 		T::AccountId,
 		Relayer<T::AccountId, RingBalance<T>>,
-		ValueQuery,
+		OptionQuery,
```

Comment:
Do we need to migrate storage if we change to OptionQuery?

I remember that it will not affect decoding, right?

Reply:
No need to.